specification/azsadmin/resource-manager/backup/Microsoft.Backup.Admin/preview/2018-09-01/Backup.json

```diff
@@ -0,0 +1,204 @@
+                    "type": "string",
+                    "readOnly": true
+                },
+                "resource": {
+                    "description": "The localized, friendly version of the resource type related to this action or operation; the resource type should match the public documentation for the resource provider.",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "operation": {
+                    "description": "The localized, friendly name for the operation. Use the name as it will displayed to the user.",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "description": {
+                    "description": "The localized, friendly description for the operation. The description will be displayed to the user. It should be thorough and concise for used in both tooltips and detailed views.",
+                    "type": "string",
+                    "readOnly": true
+                }
+            }
+        },
+        "OperationList": {
+            "description": "List of Operations",
+            "properties": {
+                "value": {
+                    "description": "Array of operations",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Operation"
+                    },
+                    "readOnly": true
+                },
+                "nextLink": {
+                    "description": "URI to the next page of operations.",
+                    "type": "string",
+                    "readOnly": true
+                }
+            }
+        },
+        "LongRunningOperationStatus": {
+            "description": "Status of a long running operation.",
+            "type": "object",
+            "properties": {
+                "code": {
+                    "description": "Operation result code.",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "message": {
+                    "description": "Description of the operation status.",
+                    "type": "string",
+                    "readOnly": true
+                }
+            }
+        }
+    },
+    "parameters": {
+        "SubscriptionIdParameter": {
+            "name": "subscriptionId",
+            "in": "path",
+            "description": "Subscription credentials that uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call.",
+            "required": true,
+            "type": "string"
+        },
+        "ApiVersionParameter": {
+            "name": "api-version",
+            "in": "query",
+            "description": "Client API version.",
+            "required": true,
+            "type": "string",
+            "default": "2018-09-01"
```

Comment:
Main difference from that in 2016-05-01 is the API version value.

Reply:
As I mentioned in email, you could have just moved this parameter to the resource themselves instead of the entire file.  Not a big deal though.

Reply:
OK I see... We added new API version in Operations endpoint in BRP so I'll keep this file this time. I'll pay attention next time.